app/src/main/graphql/com/cornellappdev/volume/schema.graphqls

```diff
@@ -67,26 +67,42 @@ type Publication {
   socials: [Social!]!
 }
 
-# ID of a Publication
-type PublicationID {
-  id: String!
+# Slug of a Publication
+type PublicationSlug {
+  slug: String!
 }
 
 type Query {
   # Returns a single <Article> via the given <id>
   getArticleByID(id: String!): Article
   # Returns a list of <Articles> via the given list of <ids>
   getArticlesByIDs(ids: [String!]!): [Article!]!
-  # Returns a list of <Articles> of size <limit>. Default <limit> is 25
-  getAllArticles(limit: Float = 25): [Article!]!
-  # Returns a list of <Articles> via the given <publicationID>
-  getArticlesByPublicationID(publicationID: String!): [Article!]!
-  # Returns a list of <Articles> via the given list of <publicationIDs>
-  getArticlesByPublicationIDs(publicationIDs: [String!]!): [Article!]!
-  # Returns a list of <Articles> via the given <slug>
-  getArticlesByPublicationSlug(slug: String!): [Article!]!
-  # Returns a list of <Articles> via the given list of <slugs>
-  getArticlesByPublicationSlugs(slugs: [String!]!): [Article!]!
+  # Returns a list of <Articles> of size <limit> with offset <offset>. Default <limit> is 25 and default <offset> is 0
+  getAllArticles(offset: Float = 0, limit: Float = 25): [Article!]!
+  # Returns a list of <Articles> of size <limit> via the given <publicationID>. Results can offsetted by <offset> >= 0.
+  getArticlesByPublicationID(
+    offset: Float = 0
+    limit: Float = 25
+    publicationID: String!
+  ): [Article!]!
+  # Returns a list of <Articles> of size <limit> via the given list of <publicationIDs>. Results offsetted by <offset> >= 0.
+  getArticlesByPublicationIDs(
+    offset: Float = 0
+    limit: Float = 25
+    publicationIDs: [String!]!
+  ): [Article!]!
```

Comment:
Why do we need getArticlesByPublicationID and getArticlesByPublicationSlug

Reply:
Gonna delete getArticlesByPublicationID, should only be doing things with slugs!